pytorch_pfn_extras/handler/__init__.py

```diff
@@ -1,6 +1,3 @@
 from pytorch_pfn_extras.handler._code_block import CodeBlock, update_parameters, forward  # NOQA
 from pytorch_pfn_extras.handler._handler import BaseHandler, Handler  # NOQA
 from pytorch_pfn_extras.handler._logic import BaseLogic, Logic, CodeBlockLogic  # NOQA
-
-# Deprecated, only imported for backward compatibility
-from pytorch_pfn_extras.handler._logic import torch_autocast  # NOQA
```

Comment:
Maybe keep it as is?

Reply:
I think it may be better to cut this at some point if possible :)

Reply:
Reverted!